contributors/design-proposals/cpu-manager.md

```diff
@@ -263,14 +267,16 @@ func (p *staticPolicy) UnregisterContainer(s State, containerID string) error {
        cpuset for all containers running in the shared pool.
 
 1. _The shared pool becomes empty._
-    1. The CPU manager adds a node condition with effect NoSchedule,
-       NoExecute that prevents BestEffort and Burstable QoS class pods from
-       running on the node. BestEffort and Burstable QoS class pods are
-       evicted from the node.
+    1. The CPU manager sets a `CPUPressure` node condition to `true`
```

Comment:
state why this is preferred versus reserving some slack cpu for scavengers?  i agree with this approach, but woudl like to explain that we dont think the complexity to support scavengers on nodes with this configuration is worth it.